src/Microsoft.DotNet.ProjectJsonMigration/ProjectContextExtensions.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Build.Construction;
+using System.Linq;
+using Microsoft.DotNet.ProjectModel;
+using System.IO;
+
+namespace Microsoft.DotNet.ProjectJsonMigration
+{
+    public static class ProjectContextExtensions
+    {
+        public static string GetProjectName(this ProjectContext projectContext)
+        {
+            return Path.GetFileNameWithoutExtension(projectContext.GetOutputPaths("_").CompilationFiles.Assembly);
```

Comment:
SaveOutputProjectRule has this same logic duplicated.  Can it call this new method instead?

[https://github.com/dotnet/cli/blob/3a567e595773b61650f2dcd1c7ec2b53b91cd7a8/src/Microsoft.DotNet.ProjectJsonMigration/Rules/SaveOutputProjectRule.cs#L13-L14](https://github.com/dotnet/cli/blob/3a567e595773b61650f2dcd1c7ec2b53b91cd7a8/src/Microsoft.DotNet.ProjectJsonMigration/Rules/SaveOutputProjectRule.cs#L13-L14)